includes/admin/payments/refunds.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Refund Details Sections
+ *
+ * @package     EDD
+ * @subpackage  Admin/Orders
+ * @copyright   Copyright (c) 2018, Easy Digital Downloads, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.0
+ */
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+/** Header ******************************************************************/
+
+/**
+ * Outputs Refund header region/notice of Refund vs. Order.
+ *
+ * @since 3.0
+ *
+ * @param \EDD\Orders\Order $refund Current Refund.
+ */
+function edd_refund_details_notice( $refund ) {
+	$order_url = edd_get_admin_url(
+		array(
+			'id'        => $refund->parent,
+			'page'      => 'edd-payment-history',
+			'view'      => 'view-order-details',
+		)
+	);
+?>
+
+<div class="edit-post-editor-regions__header">
+	<div class="edit-post-header notice-warning" style="border-left-width: 4px; border-left-style: solid;">
+
+		<div class="edit-post-header__settings">
+			<a href="<?php echo esc_url( $order_url ); ?>" class="button button-secondary" autofocus>
+				<?php esc_html_e( 'View Order', 'easy-digital-downloads' ); ?>
```

Comment:
What would you think about changing this to "View Original Order" instead? (or "Related" but I think original makes more sense in this context).